src/server.jl

```diff
@@ -440,15 +431,12 @@ function serve(fw::FileWatcher=SimpleWatcher(file_changed_callback);
         serve_file(fw, req; inject_browser_reload_script = inject_browser_reload_script, allow_cors = allow_cors)
     end
 
-    server = Sockets.listen(parse(IPAddr, host), port)
     url = "http://$(host == string(Sockets.localhost) ? "localhost" : host):$port"
     println("✓ LiveServer listening on $url/ ...\n  (use CTRL+C to shut down)")
-    HTTP.listen!(server; readtimeout=0) do http::HTTP.Stream
+    server = HTTP.listen!(host, port; readtimeout=0) do http::HTTP.Stream
```

Comment:
By changing the variable here to the actual `HTTP.Server`, we ensure a cleaner shutdown later when calling `close(server)` which will ensure all active connections being handled get closed and the underlying server socket is also closed. It seemed more robust to me in the serve + interrupt + serve again workflow.